### ISSUE TITLE
Intermittent crash in /css/css-transforms/animation/perspective-interpolation.html

### ISSUE THREAD

**Issue by jdm** [servo/servo#24239]
```
{
    "status": "OK", 
    "group": "default", 
    "message": null, 
    "stack": null, 
    "subtest": null, 
    "test": "/css/css-transforms/animation/perspective-interpolation.html", 
    "line": 35348, 
    "action": "test_result", 
    "expected": "CRASH"
}
```

**Comment by jdm** [comment]
```
  ▶ CRASH [expected OK] /css/css-transforms/animation/perspective-interpolation.html
  │ 
  │ 
  │ 
  │ 
  │ invalid parent! (thread WRRenderBackend#0, at src/libcore/option.rs:1166)
  │ stack backtrace:
  │    0: backtrace::backtrace::trace::h78e71490c44ccd31 (0x10cca64fc)
  │    1: backtrace::capture::Backtrace::new::hf7130b3c183a6059 (0x10cca5788)
  │    2: servo::main::{{closure}}::hd10c1f5a38e13522 (0x109e54347)
  │    3: std::panicking::rust_panic_with_hook::h099fe3773ce1a793 (0x10ce3ddc2)
  │    4: std::panicking::continue_panic_fmt::h8a4b76634870b383 (0x10ce3d82d)
  │    5: rust_begin_unwind (0x10ce3d729)
  │    6: core::panicking::panic_fmt::h885799c19d2b8b0c (0x10ce5df5f)
  │    7: core::option::expect_failed::hfc4846fef99c5bfe (0x10ce5dfbe)
  │    8: webrender::clip_scroll_tree::ClipScrollTree::get_relative_transform::hc887f478c42f4748 (0x10c74e70e)
  │    9: _ZN9webrender7picture18PictureUpdateState6update17hd5aae43224398938E.llvm.5198582212149417573 (0x10c858e35)
  │   10: _ZN9webrender7picture18PictureUpdateState6update17hd5aae43224398938E.llvm.5198582212149417573 (0x10c859425)
  │   11: _ZN9webrender7picture18PictureUpdateState6update17hd5aae43224398938E.llvm.5198582212149417573 (0x10c859425)
  │   12: _ZN9webrender7picture18PictureUpdateState6update17hd5aae43224398938E.llvm.5198582212149417573 (0x10c859425)
  │   13: _ZN9webrender7picture18PictureUpdateState6update17hd5aae43224398938E.llvm.5198582212149417573 (0x10c859425)
  │   14: _ZN9webrender7picture18PictureUpdateState6update17hd5aae43224398938E.llvm.5198582212149417573 (0x10c859425)
  │   15: _ZN9webrender7picture18PictureUpdateState6update17hd5aae43224398938E.llvm.5198582212149417573 (0x10c859425)
  │   16: _ZN9webrender7picture18PictureUpdateState6update17hd5aae43224398938E.llvm.5198582212149417573 (0x10c859425)
  │   17: webrender::picture::PictureUpdateState::update_all::hc5e41afd088b6e9c (0x10c8585c8)
  │   18: webrender::frame_builder::FrameBuilder::build::h7d0f96a57791b719 (0x10c753cb4)
  │   19: webrender::render_backend::Document::build_frame::hb451e6f0566ade2c (0x10c866557)
  │   20: webrender::render_backend::RenderBackend::update_document::h6f17fe401e143952 (0x10c87305c)
  │   21: webrender::render_backend::RenderBackend::run::h14685a8bf7c23509 (0x10c868261)
  │   22: std::sys_common::backtrace::__rust_begin_short_backtrace::haf7d8bfad231a0c7 (0x10c8170ce)
  │   23: _ZN3std9panicking3try7do_call17hab9c9609c598fa2fE.llvm.16585645842992353306 (0x10c81967b)
  │   24: __rust_maybe_catch_panic (0x10ce4615f)
  │   25: core::ops::function::FnOnce::call_once{{vtable.shim}}::h63ffdfc2a9bcdae8 (0x10c6d99a6)
  │   26: <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once::hb03304ace7f89c63 (0x10ce2d4ee)
  │   27: std::sys::unix::thread::Thread::new::thread_start::hd2d570f25bb299f3 (0x10ce4568e)
  │   28: _pthread_body (0x7fff5ed3f661)
  │   29: _pthread_start (0x7fff5ed3f50d)
  │ [2019-09-19T12:38:33Z ERROR servo] invalid parent!
```

**Comment by cbrewster** [comment]
Looks like a similar error to #22901 and #23438.
Should be fixed by #26063

**Comment by atouchet** [comment]
Should this be closed then?
